docs/server/routes/components/thread/demoComponents/ThreadAddButtonToMentionMenu.tsx

```diff
@@ -145,6 +145,7 @@ function TextEditorWithOpenMentionsMenu(props: TextEditorProps) {
 
 // Using the within prop allows you to isolate the replacement to specific areas.
 const REPLACEMENTS: betaV2.ReplaceConfig = {
+  // @ts-expect-error
```

Comment:
Not sure what's happening here. Had to ignore this error, otherwise the SDK wouldn't build.